integration_tests/run_tests.py

```diff
@@ -26,9 +26,9 @@
     "test_builtin_bool.py",
     "test_builtin_pow.py",
     "test_builtin_int.py",
-    "test_builtin_len.py",
+    # "test_builtin_len.py",
     "test_builtin_float.py",
-    "test_builtin_str_02.py",
+    # "test_builtin_str_02.py",
```

Comment:
Why these two test files are commented out?

Reply:
We fixed this in b07a4b31e998ca4f8a6081c221cd985367df9e1d.